src/components/WebexMember/WebexMember.scss

```diff
@@ -1,33 +1,33 @@
 .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-member {
   display: flex;
   align-items: center;
-  padding: 0.625rem 0 0.625rem 0;
 
   .details {
     flex: 1;
     min-width: 0;
-    margin-left: 0.25rem;
+    margin-left: 0.75rem;
   }
 
   .name {
     @include body-fonts;
     white-space: nowrap;
     overflow: hidden;
     text-overflow: ellipsis;
+    color: var(--wxc-member--name--color);
 
     .guest {
       color: $md-gray-70;
     }
   }
 
-  .subtext, .roles {
+  .roles {
     font-size: 0.875rem;
-    color: $md-gray-70;
+    color: var(--wxc-member--role--color);
```

Comment:
isn't this the `--wxc-secondary-text-color`? I would prefer not to have any variables dedicated to a specific webex* component. It's ok to have variables dedicated to generic components, though.